charts/rhdh/values.yaml

```diff
@@ -381,6 +381,10 @@ backstage:
               - https://instructlab.ai
               - https://quay.io
               - https://news.mit.edu
+            script-src:
+              - "'self'"
+              - "'unsafe-eval'"
```

Comment:
we can remove this, even without this it will work. I have tested it in rhdh-local.
```suggestion

```